notebooks/thoth_support_github_data.ipynb

```diff
@@ -37,6 +37,7 @@
     "from s3_communication import S3Communication\n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #3.        df = pd.DataFrame()
</pre><p>small nit, im not sure if this line is needed coz df is redefined in the immediate next line</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/open-services-group/metrics/pull/40/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>